app/models/network_port.rb

```diff
@@ -0,0 +1,17 @@
+class NetworkPort < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :cloud_tenant
+  belongs_to :cloud_network
+  belongs_to :cloud_subnets
```

Comment:
k, then here's what I think needs to happen:  we need to do a decent amount of refactoring around the models that are currently labeled `cloud_<model>`, including `cloud_network`, `cloud_subnet`, and `cloud_tenant`.

It looks like we should have `network_subnet` and `provider_tenant`.  And then, reevaluate how we're using the current `network` model.  With the possibility of splitting out physical networks from software defined networks.

But, I think part of what's holding me up here is the fact that we have these legacy `cloud`-specific models that are getting pulled into an `infra` space.  When in reality, they're more like SDN models being used by different provider types.

@matthewd, @Fryguy, @brandondunne ... does that sound about right to you too?